### PR TITLE
Upgrades ilios-common dependency

### PR DESCRIPTION
**Pull request by stopfstedt**
contains dashboard materials updates and the curriculum inventory changes.

### CODE REVIEW

File: app/components/curriculum-inventory/sequence-block-overview.hbs [hunk 1]
```diff
@@ -299,6 +299,58 @@
...
+              >
+                <select
+                  id="ending-academic-level-{{templateId}}"
+                  {{on "change" this.setEndingAcademicLevel}}
+                >
+                  {{#each (sort-by "level" this.academicLevels) as |obj|}}
+                    <option value={{obj.id}} selected={{eq obj.id this.endingAcademicLevel.id}}>{{obj.name}}</option>
+                  {{/each}}
+                </select>
+                {{#each (await (compute this.getErrorsFor "endingAcademicLevel")) as |message|}}
```
Comment (jrjohnson) [inline app/components/curriculum-inventory/sequence-block-overview.hbs:344]: Can be replaced with `<ValidationError @errors={{get-errors-for this.endingAcademicLevel}} />``

File: app/components/curriculum-inventory/sequence-block-overview.hbs [hunk 2]
```diff
@@ -299,6 +299,58 @@
...
+              >
+                <select
+                  id="starting-academic-level-{{templateId}}"
+                  {{on "change" this.setStartingAcademicLevel}}
+                >
+                  {{#each (sort-by "level" this.academicLevels) as |obj|}}
+                    <option value={{obj.id}} selected={{eq obj.id this.startingAcademicLevel.id}}>{{obj.name}}</option>
+                  {{/each}}
+                </select>
+                {{#each (await (compute this.getErrorsFor "startingAcademicLevel")) as |message|}}
```
Comment (jrjohnson) [inline app/components/curriculum-inventory/sequence-block-overview.hbs:318]: Can be replaced with `<ValidationError @errors={{get-errors-for this.startingAcademicLevel}} />``

**Review by jrjohnson — CHANGES_REQUESTED** [review]

File: app/components/curriculum-inventory/sequence-block-overview.hbs (hunk 1 above)
Comment (stopfstedt) [reply app/components/curriculum-inventory/sequence-block-overview.hbs]: :heavy_check_mark:

**Review by jrjohnson — APPROVED** [review]

File: app/components/curriculum-inventory/sequence-block-overview.hbs (hunk 2 above)
Comment (stopfstedt) [reply app/components/curriculum-inventory/sequence-block-overview.hbs]: :heavy_check_mark:

**Comment by jrjohnson** [comment]
Click tested against https://github.com/ilios/ilios/pull/3866 and looks great.

**Pull request merged**